### DISCUSSION TITLE
Integration for apollo-server-azure-functions ?

### DISCUSSION THREAD

**Discussion by MorganeLeCaignec** [dotansimha/envelop#1537] [Q&A] [answered]
Hello !
I'd love to try Envelop in a specific use-case I have, not with just Apollo Server or Azure Functions but both, with Apollo Server deployed as Azure Functions using [apollo-server-azure-functions](https://www.apollographql.com/docs/apollo-server/deployment/azure-functions/)

I'm not familiar enough yet with envelop to figure it out. I've looked at the implementation of both apollo and azure functions, the closest seems to be the apollo one but I can't get it to work. Has anyone else done this, or can someone help me?

If I understand correctly, I need to wrap the request with getEnveloped but I'm not sure how to do that with apollo-server-azure-functions.

**Comment by dotansimha** [comment, marked as answer]
Hi there! 
Please check the following thread for more info: https://github.com/apollographql/apollo-server/discussions/5541

**Reply by MorganeLeCaignec** [reply]
Hi, thank you for your response! I had previously seen that thread and the fact that Apollo's integration is partial.

My question was specifically about the integration of apollo-server-**azure-functions** that i've not been able to get working. However, trying a few things today I might switch to using another graphql server (such as helix) with azure functions.

**Reply by Urigo** [reply]
@QuentinLeCaignec for these use cases we highly recommend you to use Yoga, here is a comparison for both Helix and Apollo: https://www.the-guild.dev/graphql/yoga-server/v3/comparison
But just to make it clear, and with that said, we want to make sure all of our tools supports and integrates well with any tool, so if you can create a reproduction and make sure the issue is with Envelop and not Apollo, we would love to jump in and solve the issue

**Reply by saihaj** [reply]
you can check out Azure function with GraphQL Yoga and migration from Apollo server should be pretty straightforward https://github.com/dotansimha/graphql-yoga/tree/v2/examples/azure-function
